src/arrayMethodLastIndexOf.js

```diff
@@ -4,8 +4,22 @@
  * Implement method lastIndexOf
  */
 function applyCustomLastIndexOf() {
-  [].__proto__.lastIndexOf2 = function(searchElement, fromIndex) {
-    // write code here
+  [].__proto__.lastIndexOf2 = function(searchElement, fromIndex = this.length) {
+    if (this.length === 0 || searchElement === undefined) {
+      return -1;
+    }
+
+    let i;
+    fromIndex >= this.length ? i = this.length
```

Comment:
if you use ternary operator, you should write the element you define firstly and then its value